ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -596,6 +596,364 @@ export default {
+          };
+        } else if ((southCombatantFacing === dirNum.north && bladeSides[southCombatantBlade]) ||
+          (southCombatantFacing === dirNum.south && !bladeSides[southCombatantBlade])) {
+          // South clone cleaving inside east
+          safeZone = output.northwest();
+          adjacentZones = {
+            0: bladeValues[northCombatantBlade],
+            1: bladeValues[southCombatantBlade],
+            2: bladeValues[eastCombatantBlade],
+            3: bladeValues[westCombatantBlade],
+          };
+        } else {
+          // facing did not evaluate properly
+          safeZone = output.unknown();
+          adjacentZones = null;
+        }
+
+        let currentBrand = 0;
+        let currentTemperature = 0;
+
+        if (data.currentBrand)
+          currentBrand = data.currentBrand;
+        if (data.currentTemperature)
+          currentTemperature = data.currentTemperature;
+
+        const effectiveTemperature = currentTemperature + currentBrand;
+
+        // Calculate which adjacent zone to go to, if needed
+        let adjacentZone = null;
+        if (effectiveTemperature && adjacentZones) {
+          // Find the adjacent zone that gets closest to 0
+          const calculatedZones = Object.values(adjacentZones).map((i) =>
+            Math.abs(effectiveTemperature + i));
+
+          // Use zone closest to zero as output
+          const dirs = {
+            0: output.north(),
+            1: output.east(),
+            2: output.south(),
+            3: output.west(),
+          };
+          adjacentZone = dirs[Object.values(calculatedZones).indexOf(calculatedZones.sort((a, b) =>
+            b - a).pop())];
+        } else {
+          adjacentZone = null;
+        }
+
+        // Callout safe spot and get cleaved spot if both are known
+        // Callout safe spot only if no need to be cleaved
+        if (adjacentZone)
+          data.safeZone = output.getCleaved({ dir1: safeZone, dir2: adjacentZone });
+        else if (safeZone)
+          data.safeZone = output.safeSpot({ dir: safeZone });
+        else
+          data.safeZone = null;
+      },
+      infoText: (data, _, output) => {
+        return !data.safeZone ? output.unknown() : data.safeZone;
+      },
+      outputStrings: {
+        getCleaved: {
+          en: 'Safe Spot inside ${dir1}, Get cleaved 1 square ${dir2}',
+        },
+        safeSpot: {
+          en: 'Safe Spot: inside ${dir}',
+        },
+        unknown: {
+          en: '???',
+          de: '???',
+          fr: '???',
+          ja: '???',
+          cn: '???',
+          ko: '???',
+        },
+        north: Outputs.north,
+        northeast: Outputs.northeast,
+        east: Outputs.east,
+        southeast: Outputs.southeast,
+        south: Outputs.south,
+        southwest: Outputs.southwest,
+        west: Outputs.west,
+        northwest: Outputs.northwest,
+      },
+    },
+    {
+      id: 'DelubrumSav Avowed Blade of Entropy Collect',
```

Comment:
Can you put the collect trigger before the normal trigger?

Reply:
I defaulted to order in which the spells occurred in the encounter, but it does make it easier to read when the collect is before the trigger issuing the alert.